docs/Getting Started.md

```diff
@@ -0,0 +1,92 @@
+# Getting Started
+Welcome to Inductiva, a Python API package that simplifies running large-scale 
+physical system simulations in the cloud. Just few lines of code unleash its potential, freeing 
+you from local hardware limits and the technical hassles of scaling simulations. 
+
+### What We'll Cover:
+
+* [Inductiva API Installation]()
+* [What to read next]()
+* [Troubleshooting]()
+* [Frequently Asked Questions (FAQs)]()
+* [Useful Terminology]()
+
+## Inductiva API Installation
+
+To install the latest Inductiva package release on PyPI with pip, just open your 
+terminal and run:
+
+```
+pip install --upgrade inductiva
+```
+---
+
+> **Tip:** To ensure smooth installation and operation of our API, we recommend 
+keeping your Python package manager, Pip, up-to-date. Visit [Pip's official installation guide](https://pip.pypa.io/en/stable/installation/) for detailed instructions on updating Pip.
+
+
+Encountering issues? Don’t worry. Head over to [Troubleshooting]() to work it out.
+
+Otherwise, you're all set! You can now decide [what to read next]().
+
+## What to read next
+
+You can start experimenting with Inductiva by choosing from a variety of ready-to-use, 
```

Comment:
Let's maybe call each of these something.

This one is "Running your own simulation through Inductiva API" (see below).

Reply:
I think it's not really "choosing a simulator". You have a few simulators available -- 7 or 8 -- and then you can run *your* simulations scripts prepared for those simulators. So, we need to put the focus on the fact that users already are running simulations of their own. What they will be able to do is to run their existing simulations using the Inductiva API.

So, the very first section is about exactly that: Running your own simulation through Inductiva API

Reply:
Once we show the user how to run their existing simulations via the API we can then show how to scale them.